File: benchmark/scala-native/Global.scala
object Global {
  val ARRIVAL = 1
  val READY = 2
  val MEASURE = 3

  var time = 0.0
  
  def send(tpe: Int, dest: Proc, arrtime: Double): Unit = 
    SignalList.sendSignal(tpe, dest, arrtime)
  
}

File: benchmark/java/Global.java
public class Global{
	public static final int ARRIVAL = 1, READY = 2, MEASURE = 3;
	public static double time = 0;
	
	public static void send(int type, Proc dest, double arrtime) {
		SignalList.sendSignal(type, dest, arrtime);
	}
	
}
